NCR-Corporation/ncr-bsp-hmac
# Format Json in the Go and Javascript Languages

# Description

This PR fixes the formatting in the response data in regards to Go and Javascript.

## Testing

Please outline how to test this fix or feature. Include all steps.

1. Navigate to language directory go or js.
2. Run the program files according to their respective README.md's
3. Check to see the indented JSON format

## Ticket

MARKET-4849
